auth-web/src/components/auth/common/PADInfoForm.vue

```diff
@@ -12,8 +12,45 @@
       </p>
     </template>
     <v-form ref="preAuthDebitForm">
-      <fieldset>
-        <legend class="mb-4">Banking Information</legend>
+      <section>
```

Comment:
if its easy to do , may be make it as s separate component?we might need it in a few other places as well rite?

Reply:
Nope. Just in the one place.